{{cookiecutter.project_slug}}/README.md

```diff
@@ -8,8 +8,8 @@
 If this is your first time...
 1. [Install Docker](https://www.docker.com/)
 1. Run `pipenv lock` to generate a Pipfile.lock
-1. Run `npm install` so you have node_modules available outside of Docker
-1. `make build`
+1. (Optional) Run `cd client && npm install` so you have node_modules available outside of Docker
+1. Back in the root directory, run `make build`
```

Comment:
Is this optional again?

Reply:
Ugh, sorry, I think I got confused by a merge conflict. Will fix.